nimterop/cimport.nim

```diff
@@ -102,6 +102,9 @@ proc getToast(fullpath: string, recurse: bool = false): string =
   if gStateCT.symOverride.len != 0:
     cmd.add &"--symOverride={gStateCT.symOverride.join(\",\")} "
 
+  if gStateCT.pluginFile.nBl and gStateCT.pluginFile.fileExists():
+    cmd.add &"--pluginFile={gStateCT.pluginFile.quoteShell} "
```

Comment:
general tip:
it's always preferable to append with space ` ` BEFORE rather than after, it simplifies the contract (local responsabiilty instead of requiring cooperation from previous line); each line makes sure it keeps the cmd line valid, and doesn't require previous line to know about it; it also avoids having to add an awkward space at the end if there's not subsequent argument.

Reply:
Cleaned

Reply:
[RESOLVED]